platemapper/ordinationbuild.py

```diff
@@ -138,7 +138,7 @@ def well_convert(wl):
     df2["column"] = temp.astype(float)
 
     # create 3rd and final dataframe
-    whatever = {"sample_name": df2["sample_name"],
+    whatever = {"sample_name": df2.index,
```

Comment:
risky as we would need to ensure that this column exists. Alternatively, we use (and have to set before) the index of the dataframe